ibis/mapd/client.py

```diff
@@ -766,6 +813,23 @@ def get_schema(self, table_name, database=None):
             for col in self.con.get_table_details(table_name)
         ])
 
+    def sql(self, query):
+        """
+        Convert a SQL query to an Ibis table expression
+
+        Parameters
+        ----------
+
+        Returns
+        -------
+        table : TableExpr
+        """
+        # Get the schema by adding a LIMIT 0 on to the end of the query. If
+        # there is already a limit in the query, we find and remove it
+        limited_query = 'SELECT * FROM ({}) t0 LIMIT 1'.format(query)
```

Comment:
Any reason to use `LIMIT 1` here instead of `LIMIT 0`?

Reply:
yes, `mapd/omnisci` doesn't support `LIMIT 0`